tests/test/repository.rs

```diff
@@ -20,3 +22,19 @@ pub struct RepositorySpec {
 pub enum RepoType {
     StackableRepo,
 }
+
+pub fn setup_repository(client: &TestKubeClient) {
+    client.apply_crd(&Repository::crd());
+
+    client.apply::<Repository>(indoc! {"
+        apiVersion: stable.stackable.de/v1
+        kind: Repository
+        metadata:
+            name: integration-test-repository
+            namespace: default
+        spec:
+            repo_type: StackableRepo
+            properties:
+                url: https://raw.githubusercontent.com/stackabletech/integration-test-repo/main/
```

Comment:
Can we maybe come up with an easy way of overriding this when running tests locally? I've had to change this fairly regularly when developing a new service and running the repo locally..
Maybe have this be the default but provide functionality to override it with an env var?
I'm happy to do that myself in a later PR though, I've mostly stuck this here because we don't seem to have issues activated for this repo yet..

Reply:
I've migrated this comment into an issue - no idea why I didn't find the "issues" link earlier.